nixos/modules/services/misc/paperless.nix

```diff
@@ -252,9 +252,60 @@ in
         '';
       };
     };
+
+    exporter = {
+
+      enable = lib.mkEnableOption "regular automatic document exports";
+
+      directory = lib.mkOption {
+        type = lib.types.str;
+        default = cfg.dataDir + "/exports";
+        defaultText = "\${dataDir}/exports";
```

Comment:
Nit: Any specific reason this is export***s*** (plural) by default?

I always used `export` and was confused by this.

Reply:
I think I didn't think about this much and might have caused by the original thought that this is doing backup(s) and not export(s) ... happy to adjust this.